Missing /api/clientKeys

Hi,
when I start application with nodejs I get error Error: Cannot find module './api/clientKeys'
when I start php app, I get error require(api/clientKeys.php): failed to open stream: No such file or directory in /
Could you please investigate?
Regards,
Josip.

--- a/server/node/api/clientKeys.js
+++ b/server/node/api/clientKeys.js
@@ -1,0 +1,6 @@
+const { post } = require('request');
+const handleCallback = require('../utils/handleCallback');
+
+module.exports = (res, request) => {
+    post('/clientKeys', (err, response, body) => handleCallback({ err, response, body }, res));
+};

--- a/server/php/api/clientKeys.php
+++ b/server/php/api/clientKeys.php
@@ -1,0 +1,18 @@
+<?php
+/**
+ * Adyen Checkout Example (https://www.adyen.com/)
+ * Copyright (c) 2019 Adyen BV (https://www.adyen.com/)
+ * /originKeys Documentation: https://docs.adyen.com/api-explorer/#/CheckoutUtility/v1/originKeys
+ */
+function getClientKey() {
+	// Retrieves the clientKey from the .env file
+    $clientKey = getenv('CLIENT_KEY');
+
+    $data = [
+    	"clientKey" => $clientKey
+    ];
+
+    $result = json_encode($data);
+
+    return $result;
+}

--- a/server/php/api/clientKeys.php
+++ b/server/php/api/clientKeys.php
@@ -2,7 +2,6 @@
 /**
  * Adyen Checkout Example (https://www.adyen.com/)
  * Copyright (c) 2019 Adyen BV (https://www.adyen.com/)
- * /originKeys Documentation: https://docs.adyen.com/api-explorer/#/CheckoutUtility/v1/originKeys
  */
 function getClientKey() {
 	// Retrieves the clientKey from the .env file

